src/test/deserialization_tests.cpp

```diff
@@ -235,6 +235,9 @@ class MockTensorProtoDeserializator {
 
 MockTensorProtoDeserializatorThrowingInferenceEngine* MockTensorProtoDeserializator::mock = nullptr;
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
What is deprecated? Do we have ticket for fixing that?

Reply:
It was Throw(ov::Exception("")), The problem was we should not use throw I think. This is in tests and we expect it to happen so I do not think we need to fix this.